docs/guides/SLPDB-usage.md

```diff
@@ -0,0 +1,312 @@
+
+![Bitquery format](./images/ql3.png "Bitquery format")
+
+Our documentation on [SLPDBs MongoDB Collections & Data Schema](/tooling/slpdb/#mongodb-collections-data-schema) will get you acquainted with the different datasets you can  and their field structure. At a quick glance, 
+these are the four available DBs:
+
+- `t` stands for `tokens` and contains metadata and statistics about each token.
+- `c` stands for `confirmed` and includes any confirmed SLP transaction, valid or invalid.
+- `u` stands for `unconfirmed` and includes any unconfirmed SLP transaction, valid or invalid.
+- `g` stands for `graphs` and contains an item for each valid SLP transaction.
+
+## Execution
+
+To get started you can use two basic ways to query the existing SLPDB instances: browser and node.js.
+
+### Browser
+
+If you visit [https://slpdb.fountainhead.cash](https://slpdb.fountainhead.cash) you can edit the query on the top of the page and hit the `Run query` button on the right side. This will execute your request in 
+real time and show the result on the corresponding DB result section underneath.
+
+The notation for browser and for JavaScript are a bit different: make sure to use double quote marks instead of single ones, and remove any commas before closing curly brackets.
+
+### Node.js
+
+For a more programmatic approach, we will go through how to use Node.js to execute JavaScript snippets.
+
+First off, you'll need to use npm to install the required library `node-fetch`
+
+```bash
+> npm install node-fetch
+```
+
+After that, create a new JavaScript file in the same folder tree where you installed `node-fetch`. We called this file `SLPDB.js` and it contains the following
+
+```js
+const fetch = require('node-fetch');
+
+const fetch_retry = (url, options, n=5) => fetch(url, options).catch(function(error) {
+  if (n === 0) {
+    throw error;
+  }
+  return new Promise((resolve) => {
+    setTimeout(() => resolve(fetch_retry(url, options, n - 1)), 1000);
+  });
+});
+
+const btoa_ext = (buf) => Buffer.from(buf).toString('base64');
+
+const query = (query) => new Promise((resolve, reject) => {
+  if (! query) {
+    return resolve(false);
+  }
+  const b64 = btoa_ext(JSON.stringify(query));
+  const url = 'https://slpdb.fountainhead.cash/q/' + b64;
+
+   //console.log(url);
+
+  fetch_retry(url)
+  .then((r) => r = r.json())
+  .then((r) => {
+    if (r.hasOwnProperty('error')) {
+      reject(new Error(r['error']));
+    }
+    resolve(r);
+  });
+});
+
+//EXAMPLE QUERY
+const all_tokens = (limit=10, skip=0) => query({
+  'v': 3,
+  'q': {
+    'db': ['t'],
+    'find': {},
+    'sort': {
+      'tokenStats.approx_txns_since_genesis': -1,
+    },
+    'limit': limit,
+    'skip': skip,
+  },
+});
+
+var trythis = async function(){console.log(await all_tokens());}  
```

Comment:
var should be const? this could also be the lambda syntax like the others. ie async () => console.log(await all_tokens());